### ISSUE TITLE
PHP 8.0 Compatibility

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#153]
Make sure our plugin is compatible with PHP 8.0.  

Errors thrown here:

```
  thrown in /home/crunch10/public_html/wp-content/plugins/blog-tutor-support/includes/class-support-shortpixel.php on line 26
[07-Jul-2021 16:02:11 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "SHORTPIXEL_HIDE_API_KEY" in /home/crunch10/public_html/wp-content/plugins/blog-tutor-support/includes/class-support-shortpixel.php:26
Stack trace:
#0 /home/crunch10/public_html/wp-content/plugins/blog-tutor-support/includes/class-support-shortpixel.php(26): constant('SHORTPIXEL_HIDE...')
#1 /home/crunch10/public_html/wp-includes/class-wp-hook.php(292): NerdPress_Support_ShortPixel->is_shortpixel_bulk_optimize_set('')
#2 /home/crunch10/public_html/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters(NULL, Array)
#3 /home/crunch10/public_html/wp-includes/plugin.php(484): WP_Hook->do_action(Array)
#4 /home/crunch10/public_html/wp-settings.php(560): do_action('init')
#5 /home/crunch10/public_html/wp-config.php(88): require_once('/home/crunch10/...')
#6 /home/crunch10/public_html/wp-load.php(37): require_once('/home/crunch10/...')
#7 /home/crunch10/public_html/wp-blog-header.php(13): require_once('/home/crunch10/...')
#8 /home/crunch10/public_html/index.php(17): require('/home/crunch10/...')
#9 {main}
```

Per @ecotechie [in Slack](https://nerd-press.slack.com/archives/C01B2JJ2VSR/p1625673896220400), it's from this:
```
if ( ! isset( $options['shortpixel_bulk_optimize'] ) && ! Blog_Tutor_Support_Helpers::is_nerdpress() && @constant( 'SHORTPIXEL_HIDE_API_KEY' ) ) {
```

**Comment by blogtutor** [comment]
@ecotechie I think we fixed everything for PHP 8.0, yes?  Can we close this?
https://github.com/blogtutor/blog-tutor-support/releases/tag/1.0.3

**Comment by ecotechie** [comment]
d94750e closes this ticket.
